packages/theme/src/core/table/types.ts

```diff
@@ -95,6 +95,10 @@ export interface TableTheme {
                 };
             };
         };
+        separatorColor?: string;
+        fontFamily?: string;
```

Comment:
@mbetances805  I think `font-family` will be same for entire app. so you don’t need to extract it out inside theme. And for `font-weight` and l`etter-spacing` if you have any particular `fontVariant` in your theme then you can use that and you can keep default `fontVariant` as `h5`  in theme like below
`fontVariant: h5`